MailCatcher/app.json

```diff
@@ -0,0 +1,10 @@
+{
+    "appid": "4a5b795ca9f458e1454f96facf11c623d92d1659",
+    "name": "MailCatcher",
+    "website": "https://mailcatcher.me/",
+    "license": "MIT License",
+    "description": "MailCatcher runs a super simple SMTP server which catches any message sent to it to display in a web interface. Run mailcatcher, set your favourite app to deliver to smtp://127.0.0.1:1025 instead of your default SMTP server, then check out http://127.0.0.1:1080 to see the mail that's arrived so far.",
+    "enhanced": false,
+    "tile_background": "dark",
```

Comment:
Sure you want to make this a dark tile? Considering the icon it'd probably look better with a light one.